src/Microsoft.OpenApi.Readers/Services/OpenApiReferenceResolver.cs

```diff
@@ -176,7 +177,8 @@ private void ResolveObject<T>(T entity, Action<T> assign) where T : class, IOpen
 
             if (IsUnresolvedReference(entity))
             {
-                assign(ResolveReference<T>(entity.Reference));
+                var x = ResolveReference<T>(entity.Reference);
+                assign(x);
```

Comment:
Why this change?

Reply:
I'll revert.  I did it temporarily whilst hunting down what was taking so long.

---
In reply to: [182166389](https://github.com/Microsoft/OpenAPI.NET/pull/241#discussion_r182166389) [](ancestors = 182166389)